css/components/header.css

```diff
@@ -0,0 +1,27 @@
+header.container {
+  padding: 0;
+}
+.menu {
+  float: right;
+}
+.menu a {
```

Comment:
Teksto spalva graziai pasikeistu galim sudeti transision:
info: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Transitions/Using_CSS_transitions
mano pasibandymas tai padaryti:
transition-property: color;
  transition-timing-function: ease-in;
  transition-duration: 300ms;

Kodas dedasi prie orginalo, ne :hover kode

Reply:
pridejau, puiki mintis